LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 pre-commit dev team: Anthony Sottile, Ken Struys
+Copyright (c) 2014 Anthony Sottile
```

Comment:
the license file was copied from pre-commit/pre-commit -- adjusted this to better represent the original copyright